P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+            <name ref="#f2213001.A04" type="NICK">
+              <w lemma="Konstanze" pos="NE" xml:id="f2213001.m27.t1">Konstanze</w>
+            </name>
+            <w lemma="versuchen" pos="VVPP">versucht</w>
+            <name ref="#f2213001.A03" type="NICK">
+              <w lemma="Nasenloch" pos="NN">nasenloch</w>
+            </name>
+            <w lemma="die" pos="ART">den</w>
+            <w lemma="Wunsch" pos="NN">wunsch</w>
+            <w lemma="zu" pos="PTKZU">zu</w>
+            <w lemma="erfüllen" pos="VVINF">erfüllen</w>
+            <!-- ... -->
+          </post>
+        </egXML>
+      </p>
+      <p>In the the following version of the same chat snippet, the text strings with the nicknames
+        have been replaced by category label strings for the purpose of anonymisation. <egXML
+          xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de" source="#BIB_DCK">
+          <post modality="written" generatedBy="system" rend="color:black" synch="#f2213001a.t007"
+            type="standard" who="#f2213001a.A04" xml:id="f2213001a.m27.eg35">
+            <name ref="#f2213001a.A04" type="NICK">
+              <w pos="NE" xml:id="f2213001a.m27.t1">
+                <gap reason="anonymisation" unit="token" quantity="1"/>
+                <supplied reason="anonymisation">[_FEMALE-PARTICIPANT-A04_]</supplied></w>
+            </name>
+            <w lemma="versuchen" pos="VVPP">versucht</w>
+            <name ref="#f2213001a.A03" type="NICK">
+              <w pos="NN">
+                <gap reason="anonymisation" unit="token" quantity="1"/>
+                <supplied reason="anonymisation">[_PARTICIPANT-A04_]</supplied></w>
+            </name>
+            <w lemma="die" pos="ART">den</w>
+            <w lemma="Wunsch" pos="NN">wunsch</w>
+            <w lemma="zu" pos="PTKZU">zu</w>
+            <w lemma="erfüllen" pos="VVINF">erfüllen</w>
+            <!-- ... -->
+          </post>
+        </egXML>
+      </p>
+      <p>In the preceding example, pairs of a <gi>gap</gi> and a <gi>supplied</gi> element encode
+        the fact that some substring has been removed and replaced with another string for
+        anonymisation purposes. Note that in this example, the <gi>name</gi> and the <gi>w</gi>
+        elements and their attributes also provide some categorical information about what has been
+        removed. Using <gi>gap</gi> and <gi>supplied</gi> to record the anonymisation is especially
+        recommendable when the original name or referencing string has been
+          <soCalled>pseudonymised</soCalled>, i.e. replaced by a different referencing string of the
+        same ontological category (such as replacing the female name
+          <mentioned>Konstanze</mentioned> by the female name <mentioned>Kornelia.</mentioned>). In
+        that case, the markup would be the only place where it can be seen that a pseudonymisation
+        has been carried out, as in the following version of the example.</p>
+      <p>
```

Comment:
Some of this paragraph should perhaps be put before the example.

Reply:
There are several other things I do not like about this example:

1. It exemplifies use of `<gap>` and `<supplied>` when not really necessary — as the prose says and next example shows, those are more useful when there is an ontological equivalent swapped in for anonymization.
2. Why does one `<w>` get an ID and the other doesn’t — not _wrong_ in any way, but kinda _weird_, no?
3. Why is the part of speech of one nickname NN, but the other is NE?
4. It seems weird that the first nickname that has an anonymization string that contains “A04” and it refers to an ID that ends in “A04”, but the second nickname also has an anonymization string that contains “A04” but it refers  to an ID that ends in “A03”.
5. I would also like to reduce the complexity of the ID (and `@ref`) values just for readability.